src/tribler-core/tribler_core/utilities/tests/test_bencodecheck.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+
+from tribler_core.utilities.bencodecheck import is_bencoded
+
+
+def test_bcheck_nobytes():
+    # only bytes is valid argument type
+    with pytest.raises(ValueError, match='^Value should be of bytes type. Got: str$'):
+        is_bencoded('3:abc')
+
+def test_bcheck_empty():
```

Comment:
For the sake of readability and test speed, I would put all these tests into a single long test instead.

Reply:
Changed to a single test